### PR TITLE
chore: Supporting webdriverio v9 migration 3

### PR DESCRIPTION
**Pull request by pan-kot**
### Description

This fixes remaining incompatibility issues. The PR is a follow-up for: https://github.com/cloudscape-design/components/pull/2753

### How has this been tested?

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/collection-preferences/content-display/__integ__/pages/content-display-page.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default class ContentDisplayPageObject extends CollectionPreferencesPageO
     );
     // Using getHTML because getText returns an empty string if the live region is outside the viewport.
     // See https://webdriver.io/docs/api/element/getText/
-    return expect(liveRegion.getHTML(false)).resolves.toBe(announcement);
+    return expect(liveRegion.getHTML()).resolves.toContain(announcement);
```
Comment (cansuaa) [inline src/collection-preferences/content-display/__integ__/pages/content-display-page.ts:18]: Why this change?

**Review by cansuaa — APPROVED** [review]

File: src/collection-preferences/content-display/__integ__/pages/content-display-page.ts (hunk 1 above)
Comment (pan-kot) [reply src/collection-preferences/content-display/__integ__/pages/content-display-page.ts:18]: The getHTML() api changes. In v7 it took a flag and in v9 it takes an object.

When the flag is provided it returns the inner part of the node only. When not provided - the entire node. That's why I changed toBe to toContain. We can change this back when the webdriver dependency is updated, although that is not really necessary.
